lib/main.dart

```diff
@@ -43,19 +53,12 @@ void callbackDispatcher() {
     await Firebase.initializeApp();
 
     switch (task) {
-      case 'backgroundTracking':
-        var permission = await Geolocator.checkPermission();
-        var isBgTrackingEnabled = await UserManager.getTracking();
-
-        if (permission == LocationPermission.always && isBgTrackingEnabled){
-          var position = await Geolocator.getCurrentPosition(desiredAccuracy: LocationAccuracy.high);
-          var battery = Battery();
-          await ApiSingleton().sendFixes(position.latitude,
-                                         position.longitude,
-                                         DateTime.now().toUtc().toIso8601String(),
-                                         await battery.batteryLevel);
-          break;
-        }
+      case 'trackingTask':
+        await BackgroundTracking.trackingTask();
+        break;
+      case 'scheduleDailyTasks':
+        await BackgroundTracking.scheduleMultipleTrackingTask(5);
+        break;
     }
     return Future.value(true);
```

Comment:
@epou With this line, the workmanager should not do the automatic retry on Android, so no need to actually cancel the task if it fails https://pub.dev/packages/workmanager#work-result

Reply:
Awesome! So then, `backoffPolicy` should be removed